x-pack/plugins/ml/server/models/job_service/datafeeds.ts

```diff
@@ -160,11 +160,44 @@ export function datafeedsProvider(mlClient: MlClient) {
     }, {} as { [id: string]: string });
   }
 
+  async function getDatafeedIdByJobId(jobId: string) {
+    // if the job was created by the wizard,
+    // then we can assume it uses the standard format of the datafeedId
+    const assumedDefaultDatafeedId = `datafeed-${jobId}`;
+    try {
+      const {
+        body: { datafeeds: datafeedsResults },
+      } = await mlClient.getDatafeeds<MlDatafeedsResponse>({
+        datafeed_id: assumedDefaultDatafeedId,
+      });
+      if (Array.isArray(datafeedsResults)) {
+        if (datafeedsResults.length === 1) {
+          const datafeed = datafeedsResults[0];
+          if (datafeed.job_id === jobId) {
+            return datafeed.datafeed_id;
+          }
+        }
+      }
+    } catch (e) {
+      // if the job was doesn't use the standard datafeedId format
+      // get all the datafeeds and match it with the jobId
+      const {
+        body: { datafeeds: allDatafeedsResults },
```

Comment:
nit, i think the variable name `allDatafeedsResults`  is unnecessary

Reply:
Updated here https://github.com/elastic/kibana/pull/88898/commits/6ea550868c19746c3fe817c19195cbb492f36eba